GOVERNANCE.md

```diff
@@ -0,0 +1,132 @@
+being a role model for others.
+- Writing code &mdash; a lot of code, especially "glue code" or "grunt work";
+fixing bugs, improving processes, making judgments on dependencies, handling
+security vulnerabilities, and so on and so forth.
+- Reviewing code submitted by others &mdash; with an eye to maintainability,
+performance, code quality, and "style" (fitting in with the project).
+- Participating in design discussions, especially with regards to architecture
+or long-term vision.
+- When necessary, participate in a collective voting process to resolve
+conflicts and make decisions for which there is no clear answer.
+
+This is not an exhaustive list, nor is it intended that every maintainer does
+each and every one of these individual tasks to equal amounts. Rather this is
+only a guideline for what maintainers are expected to conceptually do.
+
+In short, Maintainers are the outwardly visible stewards of the project.
+
+#### Current list of maintainers
+
+The current list of maintainers:
+
+- Yuya Nishihara
+- Martin von Zweigbergk
+- TBD
+
+TODO: Pending on the initial process of open nominations, decided by Yuya and
+Martin, more names will be added.
+
+### Contributors
+
+We consider contributors to be active participants in the project and community
+who are *not* maintainers. These are people who might:
+
+- Help users by answering questions
+- Participating in lively and respectful discussions across various channels
+- Submit high quality bug reports
+- Submit patches or pull requests
+- Help with testing and quality assurance
+- Submit feedback about planned features, use cases, or bugs
+
+Note that we consider Contributors to all be valuable, but we *do* define them
+as people **who actively participate in the project**. Examples of things that
+would *not* make you a contributor are:
+
+- Submitting a single bug report and never returning
+- Writing blog posts or other evangelism
+- Using the software in production
+- Forking the project and maintaining your own version
+
+While these are all generally quite valuable, they don't directly contribute to
+the existing community of users where they are at, and on their own do not
+constitute "active participation".
+
+Contributors and their input is valued and their input is expected to be taken
+and respected by Maintainers; not every discussion will result in a change, but
+it is intended that every voice will be heard.
+
+## Processes
+
+For the purposes of making decisions across the project, the following processes
+are defined.
+
+### Decision Making
+
+The person proposing a decision to be made (i.e. technical, project direction,
+etc) can offer a proposal, along with a discussion timeline from 2-to-4 weeks.
+During this time, Maintainers may participate with a vote of:
+
+A) Support
+B) Rejection
+C) Abstinence
+
+A final decision can be reached upon consensus being achieved, which is defined
+as more than half of the participating votes (i.e. excluding abstaining votes.)

```

Comment:
Slightly simpler: "A final decision requires more than half of the participating votes (i.e. excluding abstaining votes.)". I mostly wanted to avoid "consensus" and the risk of the reader trying to guess if that means >=50%. We do say that we mean >50%, but the word "consensus" doesn't seem to add anything. I think the line below can also be dropped because it's just the inverse implication from this sentence (i.e. `a => b` implies `!b => !a`).